scipy/stats/tests/test_stats.py

```diff
@@ -3243,6 +3244,19 @@ def test_result_attributes(self):
         attributes = ('statistic', 'pvalue')
         check_named_results(res, attributes)
 
+    @dec.knownfailureif(True, "Tough NIST Test Cases fail")
```

Comment:
Never mind this, I've found a fix for the issue that makes all tests pass. Commit coming soon.